sdk/js/src/api/stream/stream-node.js

```diff
@@ -58,13 +58,15 @@ export default async function(payload, url) {
   }
 
   let reader = null
+  let buffer = ''
   axios({
     url,
     data: JSON.stringify(payload),
     method: 'POST',
     responseType: 'stream',
     headers: {
       Authorization,
+      'Content-Type': 'text/event-stream',
```

Comment:
Shouldn't this be an `Accept` header, since you want the response to be `text/event-stream` (and not the request)?

Reply:
Same in the other implementation

Reply:
That's true actually. While both works (setting the request to `text/event-stream` will trigger the response to be the same), I guess using `Accept` is more fitting here.